common/helpers/date-utils.js

```diff
@@ -13,6 +13,28 @@ const { DATE_FORMATS } = require('../../config/index')
 
 module.exports = {
   dateFormat: DATE_FORMATS.URL_PARAM,
+  getCurrentWeekAsRange: () => {
```

Comment:
Tests?

Reply:
Coming ;) It's still in draft but wanted to get some early eyes on the approach to make sure it made sense.